CONTRIBUTING.md

```diff
@@ -74,35 +74,63 @@ git pull upstream refs/pull/999/head
 # Pull Request Etiquette
```

Comment:
I think this section should go before 'Reviewing pull requests.' It's a little bit confusing to follow making another branch called pr_999 and then going back to the feature/999 one. Having everything grouped for what to do for one's own branch and then for someone else's work I think would be easier to follow.